src/modules/cmd_duration.rs

```diff
@@ -12,6 +12,17 @@ pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
     let mut module = context.new_module("cmd_duration");
     let config: CmdDurationConfig = CmdDurationConfig::try_load(module.config);
 
+    if cfg!(target_os = "linux") && config.tty_disabled {
```

Comment:
This block disables the entire module if not in a graphical environment, not just the notifications.

Reply:
I've moved this block to the `undistract_me` fn where the notification happens and removed the option. What i'm not sure now is how we should test this, if it is even possible. Is it ok to leave the fix without them?

Reply:
As far as in-code testing goes, I don't think there's a way to test it with our in-code tests either. Code coverage metrics are not a hard blocker to merging for us, so that's not that important.

I'm of half a mind to move the notification feature to its own module in v2.0.0, just so that we can cleanly separate the code that is functionally untestable (the stuff with notifications) from the duration stuff, which should be 100% testable, but that's not for this PR.